app/partials/app.cjsx

```diff
@@ -72,7 +70,7 @@ module.exports = React.createClass
   getEnv: ->
     reg = /\W?env=(\w+)/
     browser_env = window?.location?.search?.match(reg)
-    @state?.env || browser_env || 'staging'
+    @state?.env || browser_env[1] || 'staging'
```

Comment:
It will be the first matched substring from the regular expression, `(\w+)` in the line above. Maybe it would be better to use a more reliable method to get the environment?